<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settings</class>
 <widget class="QDialog" name="settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>182</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>182</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>182</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QSpinBox" name="time_before_lesson">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>45</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>For how many minutes before class begins to play the melody</string>
   </property>
   <property name="whatsThis">
    <string>For how many minutes before class begins to play the melody</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>140</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Before</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="on_or_off_day">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>ON\OFF calls for the this day</string>
   </property>
   <property name="whatsThis">
    <string>ON\OFF calls for the this day</string>
   </property>
   <property name="text">
    <string>ON\OFF</string>
   </property>
  </widget>
  <widget class="Line" name="line_15">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="ring_before_lesson">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Select a melody to a lesson for N minutes</string>
   </property>
   <property name="whatsThis">
    <string>Select a melody to a lesson for N minutes</string>
   </property>
  </widget>
  <widget class="Line" name="line_16">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="on_or_off_before_lesson">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>ON\OFF calls before the start of the lesson for the this day</string>
   </property>
   <property name="whatsThis">
    <string>ON\OFF calls before the start of the lesson for the this day</string>
   </property>
   <property name="text">
    <string>Time and melody before the start of the lesson</string>
   </property>
  </widget>
  <widget class="QComboBox" name="shedul_of_day">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Select a schedule for the this day</string>
   </property>
   <property name="whatsThis">
    <string>Select a schedule for the this day</string>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
   <property name="maxCount">
    <number>10</number>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
   </property>
   <property name="modelColumn">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>140</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>minutes</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="sheduleOfDay">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>ON\OFF automatic selection of the schedule for the this day</string>
   </property>
   <property name="whatsThis">
    <string>ON\OFF automatic selection of the schedule for the this day</string>
   </property>
   <property name="text">
    <string>Shedule of day:</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonPlayStop">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>138</y>
     <width>26</width>
     <height>26</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Play\Stop</string>
   </property>
   <property name="whatsThis">
    <string>Play\Stop</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../img.qrc">
     <normaloff>:/img/play.png</normaloff>:/img/play.png</iconset>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../img.qrc"/>
 </resources>
 <connections/>
</ui>
